projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx

```diff
@@ -154,6 +187,12 @@ export class DotDateRangeComponent {
         );
     }
 
+    private parseDate(strDate: string): Date {
+        const parts = strDate.split('-');
+        const newDate = new Date(parseInt(parts[0], 10), parseInt(parts[1], 10) - 1, parseInt(parts[2], 10));
+        return newDate;
+    }
```

Comment:
what happend if is not a right format date?

Reply:
the component `flatpickr` has it's own validations and when no format is correct or data type it ignores it and prints a warning on the console